### PR TITLE
eclass-writing: Update list of eclass examples.

### PR DESCRIPTION
**Pull request by ulm**

### CODE REVIEW

File: eclass-writing/text.xml [hunk 1]
```diff
@@ -27,15 +27,16 @@ Roughly speaking, there are three kinds of eclass:
 <ul>
   <li>
     Those which provide common functions which are used by many ebuilds (for
```
Comment (mgorny) [inline eclass-writing/text.xml:29]: Shouldn't that be 'these that'? Maybe sort them too, the order looks arbitrary.

Comment (ulm) [reply eclass-writing/text.xml:29]: I haven't changed the wording. It is the text from 2005, which :smile: was written by a native speaker of English.

They are currently sorted by popularity. I can make it alphabetical if you prefer.

Comment (SoapGentoo) [reply eclass-writing/text.xml:29]: Nope, this sounds correct

**Review by SoapGentoo — APPROVED** [review]

**Review by mgorny — APPROVED** [review]
That said, I think I had a different (IMO better) classification somewhere, one based on fitting exported phase functions.

**Comment by ulm** [comment]
> That said, I think I had a different (IMO better) classification somewhere, one based on fitting exported phase functions.

:shrug: The perfect is the enemy of the good.

This came about when I grepped the devmanual for `eutils`, and while at it, I've replaced the other examples as well. Rewriting the whole section isn't on my to-do list for now.

**Pull request merged**